src/ui/svgcomponent.js

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2018 The boardgame.io Authors
+ *
+ * Use of this source code is governed by a MIT-style
+ * license that can be found in the LICENSE file or at
+ * https://opensource.org/licenses/MIT.
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export class SvgComponent extends React.Component {
```

Comment:
I like that everything in `svg.js` can be easily reused across UI frameworks. How about we auto-generate `SvgComponent` wrappers for them here so that users don't have to be aware of this class (it just remains as an internal implementation adapter for React users).

Reply:
Can you elaborate on this? Where do you think the wrapper component be created? For reuse of the models, I suggest we extract the models themselves and use an UI-agnostic representation (an array somewhere).

Reply:
If I'm reading this correctly, it sounds like the user needs to actually be aware of this class in order to use the SVG models. Is this the intended usage?

```
const wheat = <SvgComponent component={Wheat}/>;
const token = <Token template={wheat}/>;
```

If so, I'm suggesting that we could just auto-generate wrappers like:

```
const WheatSVG = (props) => <SvgComponent component={Wheat} ...props/>;
const CornSVG = (props) => <SvgComponent component={Corn} ...props/>;
...
```

and use them like:

```
const token = <Token template={WheatSVG}/>
```

(swapping the names of `Wheat` and `WheatSVG` perhaps in the actual implementation).

Reply:
OK, this makes sense.